scripts/integration-test-cleanup.js

```diff
@@ -0,0 +1,93 @@
+const BbPromise = require('bluebird');
+const AWS = require('aws-sdk');
+
+const CF = new AWS.CloudFormation({ region: 'us-east-1' });
+const S3 = new AWS.S3({ region: 'us-east-1' });
+
+BbPromise.promisifyAll(CF, { suffix: 'Promised' });
+BbPromise.promisifyAll(S3, { suffix: 'Promised' });
+
+const logger = console;
+const pattern = process.env.MATCH || '(test)-[0-9]+-[0-9]+-dev.+';
+const regex = new RegExp(`^${pattern}/i`);
+
+function emptyS3Bucket(bucket) {
```

Comment:
Could we switch all the functions to an arrow syntax so that we use the new ES2015 syntax throughout the codebase? 😬